src/test/java/core/basesyntax/service/ValidatorImplTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dto.Transaction;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ValidatorImplTest {
+    private static Validator validator;
+
+    @BeforeClass
+    public static void beforeClass() {
+        validator = new ValidatorImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void validator_checkInput_sizeData_not_ok() {
+        int size = 10;
+        String[] data = new String[size];
+        validator.checkInputData(data);
+    }
+
+    @Test
+    public void validator_BalanceOperation_PositiveQuality_OK() {
+        String[] data = new String[] {"b", "apple", "13"};
+        boolean actual = validator.checkInputData(data);
+        Assert.assertTrue(actual);
+    }
+
+    @Test
+    public void validator_PurchaseOperation_PositiveQuality_OK() {
+        String[] data = new String[] {"p", "apple", "13"};
+        boolean actual = validator.checkInputData(data);
+        Assert.assertTrue(actual);
+    }
+
+    @Test
+    public void validator_SupplyOperation_PositiveQuality_OK() {
```

Comment:
you can create same test for return operation, if you created for others

Reply:
agree